frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -709,6 +736,40 @@ export function ContainerInfo(props: ContainerInfoProps) {
     );
   }
 
+  function StatusValue(props: {
```

Comment:
While I like having related components near each other in a single file, this file is more than 1000 lines and growing, maybe it's a good time to move something out of this file